src/lexer/TokenKind.ts

```diff
@@ -73,7 +73,7 @@ export enum TokenKind {
     Semicolon = 'Semicolon', // ;
     At = 'At', // @
     Callfunc = 'Callfunc', // @.
-
+    Question = 'Question', // ?
```

Comment:
I called this `Question` instead of `QuestionMark` because I'm going to name the `??` token `QuestionQuestion` for null coalescing. It felt silly to have `QuestionMark` and `QuestionQuestion`. And I don't like `QuestionMarkQuestionMark`.

Thoughts?

Reply:
Agree with this too.